io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -211,11 +211,14 @@ impl<'a, 'b: 'a> AcceptFsm for &'a mut AcceptLink<'b> {
         };
 
         // Compute the minimum batch size
-        state.transport.batch_size = state
-            .transport
-            .batch_size
-            .min(init_syn.batch_size)
-            .min(batch_size::UNICAST);
+        #[allow(clippy::unnecessary_min_or_max)]
```

Comment:
```suggestion
        // Clippy raises a warning because `batch_size::UNICAST` is currently equal to `BatchSize::MAX`.
        // However, the current code catches the cases where `batch_size::UNICAST` is different from `BatchSize::MAX`.
        #[allow(clippy::unnecessary_min_or_max)]
```